build/installers/liferay-workspace/liferay-workspace.xml

```diff
@@ -95,7 +95,122 @@
         <addDirectoryToPath>
             <insertAt>end</insertAt>
             <path>${jpmSystemPath}</path>
+            <scope>user</scope>
         </addDirectoryToPath>
+        <if>
+            <explanation>is Mac</explanation>
```

Comment:
Need to do the same thing for Linux, since it can have zshell and cshell as well.